pkg/multiTenancyPlugins/authorization/volumes.go

```diff
@@ -2,34 +2,32 @@
 package authorization
 
 import (
-	"github.com/docker/swarm/pkg/multiTenancyPlugins/headers"
-	"strings"
-	log "github.com/Sirupsen/logrus"
+	"bytes"
 	"encoding/json"
 	"errors"
-	"bytes"
-	"net/http"
-	"net/http/httptest"
+	log "github.com/Sirupsen/logrus"
 	apitypes "github.com/docker/engine-api/types"
+	"github.com/docker/swarm/pkg/multiTenancyPlugins/headers"
 	"github.com/gorilla/mux"
-	
-
+	"net/http"
+	"net/http/httptest"
+	"strings"
 )
 
-const  HOST_FS_MOUNT_NOT_ALLOWED = "Host file system mount not allowed!"
-const  VOLUME_REF_NOT_AUTHORIZED = "Volume reference Not Authorized or no such resource!"
+const HOST_FS_MOUNT_NOT_ALLOWED = "Host file system mount not allowed!"
+const VOLUME_REF_NOT_AUTHORIZED = "Volume reference Not Authorized or no such resource!"
 
 func filterVolumes(r *http.Request, rec *httptest.ResponseRecorder) []byte {
-	var response apitypes.VolumesListResponse 
+	var response apitypes.VolumesListResponse
 	if err := json.NewDecoder(bytes.NewReader(rec.Body.Bytes())).Decode(&response); err != nil {
 		log.Error(err)
 		return nil
 	}
 	var candidates []*apitypes.Volume
 	tenantId := r.Header.Get(headers.AuthZTenantIdHeaderName)
 	for _, volume := range response.Volumes {
-		if strings.HasSuffix(volume.Name,tenantId) {
-			candidates = append(candidates, volume)			
+		if strings.HasSuffix(volume.Name, tenantId) {
```

Comment:
I think in containers we agreed using label for that purpose. Can this be done here as well? Having two methods of achieving similar thing is confusing.